code/game/atoms.dm

```diff
@@ -77,6 +77,9 @@
 /atom/proc/emp_act(var/severity)
 	return
 
+/atom/proc/emp_act_console(var/severity)
```

Comment:
Ah, the relative pathing in ```mecha_control_console.dm``` threw me off. Still, it doesn't seem like this proc is ever called on a console, unless ```in_mecha()``` returns a console for some reason and is improperly casted to mecha at [line 109](https://github.com/Aurorastation/Aurora/pull/457/files#diff-e0f478c12322574b05a2ed38413b05e0R109).